### PR TITLE
release team referral emails

### PR DESCRIPTION
**Pull request by camhashemi**
@ryanpbrewster

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/payments/CreditRewardCommander.scala [hunk 1]
```diff
@@ -109,11 +109,15 @@ class CreditRewardCommanderImpl @Inject() (
       s"""
          |Your <a href="https://www.kifi.com/${referrerOrg.handle.value}">${referrerOrg.name}</a> team's referral code was used by <a href="https://www.kifi.com/${referredOrg.handle.value}">${referredOrg.name}</a>. If they upgrade to a standard plan on Kifi,
          |you'll earn a ${orgReferrerCredit.toDollarString} credit for your team. Thank you so much for spreading the word about Kifi with great teams like ${referredOrg.name}!
+         |
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/payments/CreditRewardCommander.scala]: Throw two `<br>`s in that.

**Comment by ryanpbrewster** [comment]
lgtm, nice!

File: kifi-backend/modules/shoebox/app/com/keepit/payments/CreditRewardCommander.scala [hunk 2]
```diff
@@ -109,11 +109,15 @@ class CreditRewardCommanderImpl @Inject() (
...
+         |Continue sharing your team's referral code to earn more credits: ${creditCode.value}
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/payments/CreditRewardCommander.scala]: Perhaps we could add a note about how much you'll earn?

> Continue sharing your team’s referral code to get $100 for each team who signs up for a paid plan: ${creditCode.value}

Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/payments/CreditRewardCommander.scala]: bingo

Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/payments/CreditRewardCommander.scala]: @JRuff42 ^

Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/payments/CreditRewardCommander.scala]: The credit code info has a `.credit` on it. If you pass in the whole `CreditCodeInfo` where you're currently passing in the `CreditCode` you can use it directly.

Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/payments/CreditRewardCommander.scala]: I favor "who upgrades" > "who signs up for a paid plan" to keep it a little less verbose

Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/payments/CreditRewardCommander.scala]: one step ahead of you @ryanpbrewster . `CreditCodeInfo` has been begging to get let in, I finally caved.

**Pull request merged**